examples/api/http/http.go

```diff
@@ -0,0 +1,115 @@
+package http
+
+import (
+	"context"
+	"fmt"
+	"github.com/Southclaws/fault"
+	"github.com/Southclaws/fault/fctx"
+	"github.com/Southclaws/fault/fmsg"
+	"github.com/Southclaws/fault/ftag"
+	"github.com/go-chi/chi/v5"
+	"github.com/go-chi/chi/v5/middleware"
+	"go.uber.org/zap"
+	"net/http"
+	"strings"
+	"time"
+)
+
+// Adding requestID in the fault context for logging purpose
+func RequestToFCtx(next http.Handler) http.Handler {
```

Comment:
"FCtx" does read oddly, though that's my fault for using super short package names...

```suggestion
func DecorateRequestMetadata(next http.Handler) http.Handler {
```